openmoltools/amber.py

```diff
@@ -223,7 +253,13 @@ def run_antechamber(molecule_name, input_filename, charge_method="bcc", net_char
     if frcmod_filename is None:
         frcmod_filename = molecule_name + '.frcmod'
 
-    cmd = "antechamber -i %s -fi mol2 -o %s -fo mol2 -s 2" % (input_filename, gaff_mol2_filename)
+    #Use temporary directory to do this to avoid issues with spaces in filenames, etc.
+    tempdir = tempfile.mkdtemp()
+    startdir = os.getcwd()
+    shutil.copy( input_filename, os.path.join( tempdir, 'in.mol2') )
+    os.chdir( tempdir )
```

Comment:
I'm not sure if ctrl-C would dump the user into another directory, but there could be advantages in cleaning up the temporary directory if the user aborts in the middle.